libs/extensions/src/utilities/paginate.ts

```diff
@@ -0,0 +1,57 @@
+export interface PaginatedResult<T> {
+  rows: T[];
+  meta: {
```

Comment:
@karun-rumsan  can't we use paginator from @rumsan/prisma

Reply:
@manjik-rumsan  can be use

Reply:
lets do that @karun-rumsan

Reply:
@manjik-rumsan  done